packages/compass-collection-stats/src/components/collection-stats/collection-stats.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { css } from '@mongodb-js/compass-components';
+
+import DocumentStatsItem from '../document-stats-item';
+import IndexStatsItem from '../index-stats-item';
+
+const collectionStatsStyles = css({
+  textAlign: 'right',
+  paddingRight: 0,
+  justifyContent: 'flex-end',
+  display: 'flex',
+  float: 'right',
+});
+
+const collectionStatsEmptyStyles = css({
+  display: 'none',
+});
+
+type CollectionStatsProps = {
+  documentCount?: string;
+  storageSize?: string;
+  avgDocumentSize?: string;
+  indexCount?: string;
+  totalIndexSize?: string;
+  avgIndexSize?: string;
+  isReadonly?: boolean;
+  isTimeSeries?: boolean;
+  isEditing?: boolean;
+};
+
+const CollectionStats: React.FunctionComponent<any> = ({
```

Comment:
Is it possible to use the `CollectionStatsProps` for typing to avoid the `any` here?
```suggestion
const CollectionStats: React.FunctionComponent<CollectionStatsProps> = ({
```